### PR TITLE
Ensure files created by Apache are not readable by others

### PR DESCRIPTION
**Pull request by Lex-2008**
Ticket: ENT-7948

### CODE REVIEW

**Comment by Lex-2008** [comment]
@cf-bottom build hub only

**Comment by cf-bottom** [comment]
Sure, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/7313/)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/7313/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-7313/

**Review by nickanderson — APPROVED** [review]

File: deps-packaging/apache/debian/rules [hunk 1]
```diff
@@ -37,6 +37,8 @@ install: build
 
 	$(MAKE) install DESTDIR=$(CURDIR)/debian/tmp
 
+	echo "umask 0177" >> $(CURDIR)/debian/tmp$(PREFIX)/httpd/bin/envvars
```
[the remark this answers is not in the document]
Comment (Lex-2008) [reply deps-packaging/apache/debian/rules:40]: idk, let's try!

**Comment by Lex-2008** [comment]
test build with comment: https://ci.cfengine.com/job/pr-pipeline/7320/

**Pull request merged**